### ISSUE TITLE
Upgraded to the testing branch. All ok.

### ISSUE THREAD

**Issue by tio-trom** [YunoHost-Apps/friendica_ynh#70]
I wanted to let you know that I have upgraded with: 

`sudo yunohost app upgrade friendica -u https://github.com/YunoHost-Apps/friendica_ynh/tree/testing --debug`

All works well. Here is the debug info https://paste.trom.tf/kiruremupo.yaml

The warnings seem to be related to the custom changes I made to some PHP FPM files.

I am monitoring my instance. We have over 500 users. So far it seems fine.

**Comment by tio-trom** [comment]
Maybe we should move to the latest release https://git.friendi.ca/friendica/friendica/src/branch/stable/CHANGELOG :)

**Comment by aarontrom** [comment]
I did upgrade my friendica via this command too:

> `sudo yunohost app upgrade friendica -u https://github.com/YunoHost-Apps/friendica_ynh/tree/testing --debug`

and everything works fine!

**Comment by tio-trom** [comment]
And now we have a new release https://git.friendi.ca/friendica/friendica/src/branch/stable/CHANGELOG . With that the testing is 2 releases behind and the stable ynh is way way behind :P. I am willing to test stuff if anyone is still maintaining this package.

**Comment by SpencerDub** [comment]
Does anyone know what needs to be edited to bring the `testing` branch current? I'm no developer, but I can make small edits to text files if there are simple changes to be made.

**Comment by tio-trom** [comment]
Actually there is a pull request to move this even beyond the testing, to the latest Friendica stable https://github.com/YunoHost-Apps/friendica_ynh/pull/71 - but for some reason the main developer is not merging these into the main. I have asked around I was not provided with any answers. Maybe it is a technical issue that needs to be fixed? We have tested all of these and we have hundreds of friendcia users on our instances....everything works fine. I volunteered too to help with keeping this package up to date. Seems to be as easy as changing the commit hashes from the friendica github releases like so https://github.com/YunoHost-Apps/friendica_ynh/blob/bdceafa46ae4224425089faeb10e9c7ea2c16928/scripts/_common.sh#L9-L10
